src/openqaoa-azure/requirements.txt

```diff
@@ -0,0 +1,5 @@
+azure-quantum
```

Comment:
perhaps worth confirming if we need `azure-quantum`, if `azure-quantum[qiskit]` is installed

Reply:
Just checked seems like it isnt needed if `azure-quantum[qiskit]` is installed. Will be removed.